Dockerfile

```diff
@@ -1,7 +1,31 @@
-FROM elixir:1.13.0-alpine AS build
+# Find eligible builder and runner images on Docker Hub. We use Ubuntu/Debian instead of
+# Alpine to avoid DNS resolution issues in production.
+#
+# https://hub.docker.com/r/hexpm/elixir/tags?page=1&name=ubuntu
+# https://hub.docker.com/_/ubuntu?tab=tags
+#
+#
+# This file is based on these images:
+#
+#   - https://hub.docker.com/r/hexpm/elixir/tags - for the build image
+#   - https://hub.docker.com/_/debian?tab=tags&page=1&name=bullseye-20220801-slim - for the release image
+#   - https://pkgs.org/ - resource for finding needed packages
+#   - Ex: hexpm/elixir:1.13.4-erlang-25.0.4-debian-bullseye-20220801-slim
+#
+ARG ELIXIR_VERSION=1.13.4
+ARG OTP_VERSION=25.0.4
+ARG DEBIAN_VERSION=bullseye-20220801-slim
+
+ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
+ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"
+
+FROM ${BUILDER_IMAGE} as builder
+
+ARG DEPLOY_DOMAIN="https://elixirschool.com"
```

Comment:
The generating of rss and sitemap are now done at build time instead of runtime. This speeds up starting the container and fixes issues with the file permissions. It's good to have read only containers, m-kay. Anyway, this build arg can adjust the domain that is used for generating those two files.